packages/contracts-bedrock/test/L1/OptimismPortal.t.sol

```diff
@@ -160,6 +261,8 @@ contract OptimismPortal_Test is CommonTest {
     ///      This places an upper bound on unsafe blocks sent over p2p.
     function test_depositTransaction_largeData_reverts() external {
         uint256 size = 120_001;
+
+        // Call minimumGasLimit(0) before vm.expectRevert to ensure vm.expectRevert is for depositERC20Transaction
         uint64 gasLimit = optimismPortal.minimumGasLimit(uint64(size));
```

Comment:
not sure what this comment means, we are calling `minimumGasLimit(120001)`

Reply:
If the call is inlined, then it will expect that `minimumGasLimit` reverts